web/server.js

```diff
@@ -1 +1,51 @@
-comming soon
\ No newline at end of file
+var express = require('express');
+var app = express();
+var bodyparser = require('body-parser');
+var fs = require('fs');
+var open = require('open');
+var exec_cmd = require('./exec_cmd');
+
+app.set('view engine', 'ejs');
+app.use(express.static('public'));
+app.use(bodyparser.urlencoded({extended : false}));
+
+var cmd_file = 'cmd.txt';
+const port = 3000;
+
+app.get('/', (req, res) => {
+    res.render('bteq', {data : ""});
+});
+
+app.post('/', (req, res) => {
+    var {'logon' : log, 'set-width' : wid, 'select' : sel} = req.body;
+    var logon = undefined;
```

Comment:
can you use **let** instead of **var**? Also, please read why it's more preferable.